include/ppx/xr_component.h

```diff
@@ -95,6 +96,21 @@ struct XrComponentResolution
     uint32_t height = 0;
 };
 
+class XrCamera : public Camera
+{
+public:
+    XrCamera() = default;
+
+    ppx::CameraType GetCameraType() const final { return ppx::CAMERA_TYPE_UNKNOWN; }
```

Comment:
Should this be CAMERA_TYPE_PERSPECTIVE or a new type?

Reply:
It's a perspective projection, however it is different from `PerspCamera`, as far as I can see this attribute of camera is unused so I put it as `CAMERA_TYPE_UNKNOWN` and we can figure out what exactly it should be when we need the camera type.